docs/kubernetes/config_examples/f5-k8s_multi-tenant-1.yaml

```diff
@@ -0,0 +1,109 @@
+# Deploy a BIG-IP Controller instance that creates objects for all namespaces
+# in the "ose-cluster" partition on the BIG-IP system
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: k8s-bigip-ctlr-deployment
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: k8s-bigip-ctlr
+      labels:
+        app: k8s-bigip-ctlr
+    spec:
+      serviceAccountName: bigip-ctlr-serviceaccount
+      containers:
+        - name: k8s-bigip-ctlr-1
+          image: "f5networks/k8s-bigip-ctlr:1.3.0"
+          env:
+            - name: BIGIP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: username
+            - name: BIGIP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: bigip-login
+                  key: password
+          command: ["/app/bin/k8s-bigip-ctlr"]
+          args: [
+            "--bigip-partition=ose-cluster",
+            "--bigip-url=10.190.24.171",
+            "--bigip-username=$(BIGIP_USERNAME)",
+            "--bigip-password=$(BIGIP_PASSWORD)",
+            "--use-secrets=true",
+            "--pool-member-type=cluster",
+            ...
```

Comment:
We should only provide example manifests that work 100% if a customer is to download them and then try to deploy it. Remove line 39 ellipsis as it's not needed.